test/unit-tests/openBrowser.test.js

```diff
@@ -11,6 +12,23 @@ describe(' opens browser successfully',()=>{
         });
     });
 
+
+    test('openBrowser should initiate the CRI client object',  ()=>{
+
+        return openBrowser().then(() => {
+            expect(client).not.toBeNull();
+        });
+    });
+
+
+    test('openBrowser should initiate the CRI client object with Browser domain',  ()=>{
+
+        return openBrowser().then(() => {
+            expect(client.Browser).not.toBeNull();
```

Comment:
`client.Browser` is CRI's internal domain.. if client is initialised it should be there always we don't have to verify this.